charts/incubator/amd-automatic-music-downloader/values.yaml

```
@@ -0,0 +1,29 @@
+image:
+  repository: tccr.io/truecharts/amd-automatic-music-downloader
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:9d0cf8206966456e065a1ded124b28dd0fb6172c5e6ded082e10788cfcadb24f
+
```

```suggestion
  repository: tccr.io/truecharts/automatic-music-downloader
  pullPolicy: IfNotPresent
  tag: latest

```
https://github.com/truecharts/containers/pull/128 Once this merged and made public, you can see its new digest in packages section of https://github.com/truecharts/containers/